### PR TITLE
Report unnumbered TODOs

### PR DESCRIPTION
**Pull request by maltekliemann**
Fixes part of zeitgeistpm/runtime-audit-1#1.

### CODE REVIEW

**Comment by maltekliemann** [comment]
There last TODO is fixed by #553, so this is blocked until #553 is merged.

**Review by Chralt98 — COMMENTED** [review]
This PR ensures with rustfmt to let the workflow fail if a `TODO` comment in the code is not addressed with a numbered issue like `// TODO(#123) Implement...`. An example proof of this is [here](https://github.com/zeitgeistpm/zeitgeist/runs/5978273210?check_suite_focus=true).

Apart from that, in the [runtime audit](https://github.com/zeitgeistpm/runtime-audit-1/issues/1) the last two references don't link to the TODO's. The TODO's were already removed.

File: zrml/prediction-markets/src/lib.rs [hunk 1]
```diff
@@ -781,7 +779,7 @@ mod pallet {
         pub fn global_dispute(origin: OriginFor<T>, market_id: MarketIdOf<T>) -> DispatchResult {
             let _sender = ensure_signed(origin)?;
             let _market = T::MarketCommons::market(&market_id)?;
-            // TODO: implement global disputes
+            // TODO(#489): Implement global disputes!
```
[the remark this answers is not in the document]
Comment (Chralt98) [reply zrml/prediction-markets/src/lib.rs:782]: Not better remove the `global_dispute` function and paste [this link](https://github.com/zeitgeistpm/zeitgeist/blob/386b43dd3a7025b8e76bf9cfbf310675bb114af8/zrml/prediction-markets/src/lib.rs#L774-L784) reference [here](https://github.com/zeitgeistpm/zeitgeist/blob/386b43dd3a7025b8e76bf9cfbf310675bb114af8/zrml/prediction-markets/src/lib.rs#L33)?

File: zrml/prediction-markets/src/lib.rs [hunk 2]
```diff
@@ -135,6 +132,7 @@ mod pallet {
             origin: OriginFor<T>,
             market_id: MarketIdOf<T>,
         ) -> DispatchResultWithPostInfo {
+            // TODO(#486)
```
[the remark this answers is not in the document]
Comment (maltekliemann) [reply zrml/prediction-markets/src/lib.rs:135]: I think we shouldn't make a habit of storing which PR fixes which error in the code. The way I see it, the TODO refers to GitHub, which then refers to which PRs handle the issue. What do you think?

File: zrml/orderbook-v1/src/lib.rs [hunk 3]
```diff
@@ -4,8 +4,6 @@
 //!
 //! ## Overview
 //!
-//! TODO
```
[the remark this answers is not in the document]
Comment (maltekliemann) [reply zrml/orderbook-v1/src/lib.rs:7]: Yeah, just a generic TODO.

File: zrml/prediction-markets/src/lib.rs [hunk 4]
```diff
@@ -781,7 +779,7 @@ mod pallet {
         pub fn global_dispute(origin: OriginFor<T>, market_id: MarketIdOf<T>) -> DispatchResult {
```
[the remark this answers is not in the document]
Comment (maltekliemann) [reply zrml/prediction-markets/src/lib.rs:776]: Very good point. I've remove the extra work for now.

File: zrml/prediction-markets/src/lib.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (maltekliemann) [reply zrml/prediction-markets/src/lib.rs:778]: Yeah, even better. Just get rid of it for the time being. :thumbsup:

File: zrml/prediction-markets/src/lib.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Chralt98) [reply zrml/prediction-markets/src/lib.rs:134]:
```suggestion
            // TODO(#486)
```

I thought about using the [number 43](https://github.com/zeitgeistpm/runtime-audit-1/issues/43), but it does not point to the Zeitgeist node repo. So, I agree, that we should do a reference to the related issue, which then should always point to the pull request.

**Review by Chralt98 — COMMENTED** [review]
Looks fine to me.

**Pull request merged**